### PR TITLE
Add breadcrumbs to catalog and wizard views

### PR DESCRIPTION
**Pull request by glekner**
## 📝 Description

Add breadcrumbs to catalog and wizard views

## 🎥 Demo

Catalog:
Screen Shot 2022-04-11 at 20 17 00

Customize:
Screen Shot 2022-04-11 at 19 41 30

Wizard:
Screen Shot 2022-04-11 at 20 16 42

### CODE REVIEW

File: src/views/catalog/customize/components/CustomizeVirtualMachineHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import { VirtualMachineModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+
+export const CustomizeVirtualMachineHeader: React.FC<{ namespace: string }> = React.memo(
+  ({ namespace }) => {
+    const { t } = useKubevirtTranslation();
+    const history = useHistory();
+    const ref = VirtualMachineModelGroupVersionKind;
```
Comment (avivtur) [inline src/views/catalog/customize/components/CustomizeVirtualMachineHeader.tsx:12]: redundant, use `VirtualMachineModelRef` directly

File: src/views/catalog/customize/components/CustomizeVirtualMachineHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    return (
+      <div className="co-m-nav-title co-m-nav-title--breadcrumbs">
+        <Breadcrumb>
+          <BreadcrumbItem>
+            <Button
+              variant="link"
+              isInline
+              onClick={() =>
+                history.push(
+                  `/k8s/ns/${namespace || 'default'}/${ref.group}~${ref.version}~${ref.kind}`,
```
Comment (avivtur) [inline src/views/catalog/customize/components/CustomizeVirtualMachineHeader.tsx:23]: like this :)
`/k8s/ns/${namespace || 'default'}/${VirtualMachineModelRef}`

File: src/views/catalog/templatescatalog/components/TemplatesCatalogPageHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+
+export const TemplatesCatalogPageHeader: React.FC<{ namespace: string }> = React.memo(
+  ({ namespace }) => {
+    const { t } = useKubevirtTranslation();
+    const history = useHistory();
+    const ref = VirtualMachineModelGroupVersionKind;
```
Comment (avivtur) [inline src/views/catalog/templatescatalog/components/TemplatesCatalogPageHeader.tsx:19]: here too :)

File: src/views/catalog/wizard/components/WizardHeader.tsx [hunk 4]
```diff
@@ -1,23 +1,86 @@
...
+  Title,
+} from '@patternfly/react-core';
 
-export const WizardHeader: React.FC = () => {
+export const WizardHeader: React.FC<{ namespace: string }> = React.memo(({ namespace }) => {
   const { t } = useKubevirtTranslation();
+  const { tabsData } = useWizardVMContext();
+  const history = useHistory();
+
+  const ref = VirtualMachineModelGroupVersionKind;
```
Comment (avivtur) [inline src/views/catalog/wizard/components/WizardHeader.tsx:22]: same :)

**Review by upalatucci — APPROVED** [review]
/lgtm

**Pull request merged**